ResearchKit/Consent/ORKConsentLearnMoreViewController.h

```diff
@@ -38,6 +38,7 @@ NS_ASSUME_NONNULL_BEGIN
 @interface ORKConsentLearnMoreViewController : UIViewController
 
 - (instancetype)initWithHTMLContent:(NSString *)content;
+- (instancetype)initWithContentRequest:(NSURLRequest *)contentRequest;
```

Comment:
Right, but would there ever be a need to specify any parameter of the request other than the URL?